core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -108,6 +108,7 @@ class ControllerChannelManager(controllerContext: ControllerContext, config: Kaf
     val messageQueue = new LinkedBlockingQueue[QueueItem]
     debug("Controller %d trying to connect to broker %d".format(config.brokerId, broker.id))
     val brokerNode = broker.getNode(config.interBrokerListenerName)
+    val logContext = new LogContext("[ControllerChannelManager brokerId=" + config.brokerId.toString + "] ")
```

Comment:
Maybe this could be:

```scala
s"[ControllerChannelManager controllerId=${config.brokerId}, targetBrokerId=${brokerNode.idString}] "
```

It's a bit long though, we could potentially shorten `ControllerChannelManager` to `Controller` and replace `controllerId` with `id`. @hachikuji, what do you think?

Reply:
Yeah, I'd vote to shorten both. Seems not much room for confusion.